server.py

```diff
@@ -152,12 +175,14 @@ def get_product_data():
         reader = csv.DictReader(csvfile)
         for row in reader:
             prod_data[int(row['id'])] = {   'id':int(row['id']), 'name':row['name'], 'category':row['category'], \
-                                            'discontinued': row.get('discontinued', False)   }
+                                            'discontinued': row.get('discontinued', False), \
+                                            'price': row['price']   }
```

Comment:
By default, `'price': row['price'] ` will make the 'price' attribute into a string.

It should be casted into an int or a decimal, e.g.  `'price': int(row['price']) `

Note: the _sampleSchema_products.csv_ file may have to updated to ensure that all values in the 'price' column are valid numerical values.